MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Commit in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the commit feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: ug
+---
+
+## Commit mode
+
+The [CommitMode] determines when the value should be committed to the data object.
```

Comment:
The [CommitMode] determines when the value should be committed to the underlying data object  to allow users commit only correct values.

Reply:
content changed